config/config.toml.example

```diff
@@ -455,6 +455,9 @@ enable-internal-query = false
 # max number of statements kept in memory.
 max-stmt-count = 200
 
+# max number of evicted info kept in memory.
+max-evicted-count = 48
+
```

Comment:
No need to add a new config item.

Reply:
Keep sync with statesment_summary config?